File: JavaScript/exercicio-07/solucao.js
const prompt = require("prompt-sync")();
const readline = require("readline-sync");

function main() {

   function tercaParte_dobro() {

      const numero = parseFloat(prompt("Digite um número: " ));
      const dobro = numero * 2;
      const tercaParte = numero / 3;
      
      console.log(`O Dobro de ${numero} é ${dobro} e a terca parte de ${numero} é ${tercaParte.toFixed(2)}`);
          
   }
   
   tercaParte_dobro();

}

main();
readline.question('Pressione Enter para sair ...');

File: C/exercicio-07/solucao.c
#include <stdio.h>

void dobroTercaParte() {
  float num, dobro, tercaParte;

  printf("Digite um número: ");
  scanf("%f", &num);

  dobro = num * 2;
  tercaParte = num / 3;

  printf("O dobro de %f é igual a %f", num, dobro);
  printf("A terca parte de %f é igual a %f", num, tercaParte);

}

int main() {
  dobroTercaParte();
  return 0;
}